build/yaml/steps/codeanalysis/csharp.yaml

```diff
@@ -9,7 +9,7 @@ steps:
   inputs:
     continueOnError: true
     rulesetName: Custom
-    msBuildArchitecture: x86
+    msBuildArchitecture: x64
```

Comment:
>x64 [](http://example.com/codeflow?start=25&length=3)

I'm assuming the reason for changing this is performance?

Reply:
It might not be necessary to change it, but the installation of dotnet from the UseDotNet tasks with install for the native architecture of the OS, which is x64 for the build machines. I ran the static analysis build and it seemed to work fine.